pkg/embedded/embedded.go

```diff
@@ -158,6 +163,46 @@ func runK8s(ctx context.Context, kubeConfig string, args []string) {
 			logrus.Errorf("Failed to build client: %v", err)
 			return
 		}
+	} else {
+		if _, err := os.Stat("/etc/kubernetes/ssl/config.conf"); os.IsNotExist(err) {
+			save, err := os.Create("/etc/kubernetes/ssl/config.conf")
```

Comment:
is the file name a name k8s recognizes by default? can you put a ref to the doc here?

Reply:
it's just path to any file that contains the encryption config, I chose this path because it was also present in our packaging for `/etc/kubernetes/ssl/certs` and `/etc/kubernetes/ssl/certs/serverca`